statsd/tests/test_statsd.py

```diff
@@ -0,0 +1,78 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import os
+import sys
+import time
+import pytest
+import requests
+import subprocess
+from datadog_checks.utils.common import get_docker_hostname
+from datadog_checks.statsd import StatsCheck
+
+CHECK_NAME = 'statsd'
+HOST = get_docker_hostname()
+PORT = 8126
+HERE = os.path.dirname(os.path.abspath(__file__))
+URL = "http://{}:{}".format(HOST, PORT)
+
+METRICS = [
+    'statsd.graphite.flush_length',
+    'statsd.messages.last_msg_seen',
+    'statsd.counters.count',
+    'statsd.graphite.last_exception',
+    'statsd.gauges.count',
+    'statsd.messages.bad_lines_seen',
+    'statsd.graphite.flush_time',
+    'statsd.graphite.last_flush',
+    'statsd.uptime',
+    'statsd.timers.count'
+]
+
+SERVICE_CHECKS = [
+    'statsd.is_up',
+    'statsd.can_connect'
+]
```

Comment:
Can we get these service check names directly from the `StatsCheck` file? That way we don't have to hardcode them in.